library/src/main/java/link/fls/swipestack/SwipeHelper.java

```diff
@@ -92,12 +97,24 @@ public boolean onTouch(View v, MotionEvent event) {
                 return true;
 
             case MotionEvent.ACTION_UP:
+                float x2 = event.getX();
+                float y2 = event.getY();
+                dx = x2 -x1;
+                dy = y2 -y1;
+                float MAX_CLICK_DISTANCE = 0.5f;
```

Comment:
I think this might be better defined as a constant on the class (private static final) for better organization. I also find that a value a little above this works better.